hnn_core/tests/test_compare_hnn.py

```diff
@@ -36,7 +36,7 @@ def test_hnn_core():
 
     # Test spike type counts
     spiketype_counts = {}
-    for spikegid in net.spikegids[0]:
+    for spikegid in net.spikes.gids[0]:
```

Comment:
I'm sorry if this is obvious, but what's the difference between `spikes.gids` and `spikes._gids`?

Reply:
No difference. It's a safety mechanism so that the user doesn't modify `spikes.gids`.